site/src/components/RoleSelect/RoleSelect.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { screen } from "@testing-library/react"
+import {
+  assignableRole,
+  MockAssignableSiteRoles,
+  MockAuditorRole, MockMemberRole,
+  MockOwnerRole, MockTemplateAdminRole,
+  MockUserAdminRole,
+  render
+} from "../../testHelpers/renderHelpers"
```

Comment:
```suggestion
} from "testHelpers/renderHelpers"
```
We're trying to favor absolute paths when we're not in the same directory as whatever we're importing.